docs/src/main/paradox/user/lang/go/index.md

```diff
@@ -0,0 +1,16 @@
+# Go
+
+Cloudstate offers an idiomatic, annotation based Go support library for writing stateful services.
```

Comment:
"idiomatic, annotation based Go support" ^_^"

- [ ] remove the annotation mention.